.circleci/config.yml

```diff
@@ -0,0 +1,51 @@
+version: 2.1
+orbs:
+  ruby: circleci/ruby@0.1.2
+
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.6.3-stretch-node
+    executor: ruby/default
+    steps:
+      - checkout
+      - run:
+          name: Install/Upgrade Bundler
+          command: gem install bundler
+      - run:
+          name: Which bundler?
+          command: bundle -v
+      - ruby/bundle-install
+      - run:
+          name: Set up CodeClimate
+          command: |
+            curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+            chmod +x ./cc-test-reporter
+            ./cc-test-reporter before-build
+      - run:
+          name: Validate API specification
+          command: |
+            sudo npm install -g openapi-enforcer-cli
+            result=$(openapi-enforcer validate openapi.yml)
+            [[ $result =~ "Document is valid" ]] && {
+            echo "Validation good"
+            exit 0
+            } || {
+            echo $result
+            exit 1
+            }
+      - run:
+          name: Check styles using rubocop
+          command: bundle exec rubocop
+      - run:
+          name: Setup Code Climate test-reporter
+          command: |
+            curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
+            chmod +x ./cc-test-reporter
+            ./cc-test-reporter before-build
```

Comment:
Have you already set `CC_TEST_REPORTER_ID` in the CircleCI env for DIA?

Reply:
Yes, I did that

Reply:
thank you!